Source/WebKit/WebProcess/Cache/WebCacheStorageConnection.cpp

```diff
@@ -67,7 +69,9 @@ void WebCacheStorageConnection::open(const WebCore::ClientOrigin& origin, const
 
 void WebCacheStorageConnection::remove(WebCore::DOMCacheIdentifier cacheIdentifier, WebCore::DOMCacheEngine::RemoveCacheIdentifierCallback&& callback)
 {
-    connection().sendWithAsyncReply(Messages::NetworkStorageManager::CacheStorageRemoveCache(cacheIdentifier), WTFMove(callback));
+    auto iterator = m_origins.find(cacheIdentifier);
+    RELEASE_ASSERT(iterator != m_origins.end());
+    connection().sendWithAsyncReply(Messages::NetworkStorageManager::CacheStorageRemoveCache(iterator->value, cacheIdentifier), WTFMove(callback));
```

Comment:
We are adding origin to each IPC, which makes IPC messages quite bigger.
The final plan is probably to send an identifier instead, which could be retrieved from open IPC result here.
Maybe you could document this in the change log, since moving to identifier might make the itch even bigger.